content/docs/introduction/glossary.md

```diff
@@ -32,28 +32,33 @@ pull metrics from other systems and expose the metrics to Prometheus.
 ### Collector
 
 A collector is a part of an exporter that represents a set of metrics. It may be
-a single metric as part of direct instrumentation, or many metrics if it is pulling
-metrics from another system.
+a single metric if it is part of direct instrumentation, or many metrics if it is pulling metrics from another system.
 
 ### Direct instrumentation
 
-Direct instrumentation is when instrumentation is added inline as part the source code
+Direct instrumentation is instrumentation added inline as part the source code
 of a program.
 
 ### Exporter
 
 An exporter is a binary that exposes Prometheus metrics, commonly by converting
 metrics that are exposed in a non-Prometheus format into a format Prometheus supports.
 
+### Instance
+
+An individually scraped target, usually corresponding to a single process.
```

Comment:
An individually scraped target is a target. A single instance label might correspond to multiple targets in niche use cases.

Reply:
So I lifted these from the Jobs and Instances Concept page. is that not accurate?

Reply:
No, it's not 100% accurate. An instance is what can be scraped, a target is the one or more ways we actually scrape it.

Reply:
Would you like me to try to update both the Concepts page and here? Happy to leave it and remove too.